test/OpenTelemetry.Exporter.Geneva.Tests/OtlpProtobufMetricExporterTests.cs

```diff
@@ -0,0 +1,237 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using System.Linq;
+using Google.Protobuf;
+using Google.Protobuf.Collections;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using Xunit;
+using OtlpCollector = OpenTelemetry.Proto.Collector.Metrics.V1;
+using OtlpCommon = OpenTelemetry.Proto.Common.V1;
+
+namespace OpenTelemetry.Exporter.Geneva.Tests;
+
+public class OtlpProtobufMetricExporterTests
+{
+    [Fact]
+    public void LongCounterSerializationSingleMetricPoint()
+    {
+        using var meter = new Meter(nameof(this.LongCounterSerializationSingleMetricPoint), "0.0.1");
+        var longCounter = meter.CreateCounter<long>("longCounter");
+        var exportedItems = new List<Metric>();
+        using var inMemoryReader = new BaseExportingMetricReader(new InMemoryExporter<Metric>(exportedItems))
+        {
+            TemporalityPreference = MetricReaderTemporalityPreference.Delta,
+        };
+
+        using var meterProvider = Sdk.CreateMeterProviderBuilder()
+            .AddMeter(nameof(this.LongCounterSerializationSingleMetricPoint))
+            .AddReader(inMemoryReader)
+            .Build();
+
+        longCounter.Add(123, new("tag1", "value1"), new("tag2", "value2"));
```

Comment:
Add a test case for testing negative long numbers and ulong numbers as well.

Reply:
Added for negative number. Planning to do ulong and negative attribute value in follow up.